tools/slang-cpp-parser/unit-test.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Ideally this should have been created with `git mv`, so that history is retained.

Reply:
I used `git mv` for all files. This one is small and changed a lot so git can't track that. Performing multiple commits should solve it but would break again once this PR gets squash merged.

Reply:
yeah, fwiw `git mv` doesn't actually do any special tracking, it's just short for normal `mv`, untrack old file, track new file. the detects file moves purely from text similarity.